src/contextMenu.component.ts

```diff
@@ -23,10 +23,10 @@ export interface ILinkConfig {
   html: (item: any) => string;
 }
 export interface MouseLocation {
-  left?: string;
-  marginLeft?: string;
-  marginTop?: string;
-  top?: string;
+  left?: number;
+  marginLeft?: number;
+  marginTop?: number;
+  top?: number;
 }
```

Comment:
What is your reason for changing these to numbers?  I'm not necessarily against it, just curious.